src/idiom286ex.fz

```diff
@@ -0,0 +1,8 @@
+ex286 is
```

Comment:
use `=>` and omit `_ :=`

Reply:
or actually you can also just keep the `is`. Does not really matter in this case.